Week2/git_push.py

```diff
@@ -20,18 +19,25 @@ def create_pull_request(project_name, repo_name, title, description, head_branch
     r = requests.post(
         git_pulls_api,
         headers=headers,
-        data=json.dumps(payload))
+        data=json.data(payload))
```

Comment:
When I running the code there is a AttributeError: module 'json' has no attribute 'data'

Reply:
what I meant was `data=payload`